src/modules/attributes.ts

```diff
@@ -1,14 +1,6 @@
 import { VNode, VNodeData } from '../vnode';
 import { Module } from './module';
 
-// because those in TypeScript are too restrictive: https://github.com/Microsoft/TSJS-lib-generator/pull/237
-declare global {
-  interface Element {
-    setAttribute(name: string, value: string | number | boolean): void
-    setAttributeNS(namespaceURI: string, qualifiedName: string, value: string | number | boolean): void
-  }
-}
-
```

Comment:
If you look below, you may find the reason for this re-declare. What I did not realize is that it would be re-declared for the users, as well. That's global pollution. It's a no-no.

So, you may ask why can this re-declaration be removed without the calls to `Element.setAttribute` below failing to type check? Good question. That's because this is still re-declared, even though this is removed. See, this is not the only place where this is re-declared. The other place is by importing our own project under a different name, in the benchmark test:

https://github.com/snabbdom/snabbdom/blob/ef719f41ca9eb5b9c08a5d2aa2790f8f3280a54f/src/benchmark/core.ts#L6
https://github.com/snabbdom/snabbdom/blob/ef719f41ca9eb5b9c08a5d2aa2790f8f3280a54f/package.json#L45

In light of that, you could claim that this PR is not complete, because the calls below should not pass type checking only due to the global pollution of the last published version of our package. Good point. Then, the more complete PR would also take care to receive the compiler's agreement on those calls. But I prefer to wait until this change is in a published package and then update that import so that the lack of the global pollution would actually assist me in making that more complete change with the correct type checking. It's kind of awkward but harmless and convenient.

Reply:
If we'd not remove that we should add a describing comment, IMHO.